frontend/packages/ceph-storage-plugin/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@console/ceph-storage-plugin",
+  "version": "0.0.0-fixed",
+  "description": "Ceph Storage - Persistent storage for kubernetes ",
```

Comment:
Nit-pick: should be Kubernetes (not kubernetes) and without trailing space.

Reply:
@vojtechszocs done